skimage/segmentation/_quickshift.pyx

```diff
@@ -14,7 +14,7 @@ from ..util import img_as_float
 from ..color import rgb2lab
 
 
-def quickshift(image, ratio=1., float kernel_size=5, max_dist=10,
+def quickshift(image, ratio=1.0, kernel_size=5, max_dist=10,
```

Comment:
`kernel_size` is used in a nested for-loop in this function.  It seems like we should precompute the value used in that loop and store it.